src/Service/BaseImporter.php

```diff
@@ -30,6 +32,34 @@ protected function sanitizeText(string $str): ?string
         return $str;
     }
 
+    /**
+     * @param array<int,mixed>|null $list
+     *
+     * @return string|null
+     */
+    protected function convertList(?array $list): ?string
+    {
+        if ($list) {
+            return implode(', ', $list);
+        }
+
+        return '';
+    }
+
+    /**
+     * @param $obj
+     *
+     * @return string|null
+     */
+    protected function convertLink(?object $obj): ?string
+    {
+        if ($obj && $obj->Url && $obj->Description) {
+            return '<a href="'.$obj->Url.'">'.$obj->Description.'</a>';
```

Comment:
Consider adding a call to `htmlspecialchars` here:

```suggestion
            return '<a href="'.htmlspecialchars($obj->Url).'">'.$obj->Description.'</a>';
```

It's probably not needed, but it's better to be safe than sorry.